ROBOT:
  hardware_interface:
     ids:
       - "1"
       - "2"
     velocities:
       - 0.05
       - 0.05
     homing_offsets:
       - -10.0
       - -5.0
  mode: real
  port: auto
  homing_on_start: 1
